docs/cloud-adoption/decision-guides/encryption/overview.md

```diff
@@ -33,24 +33,25 @@ When planning a cloud migration, the following table describes how you can store
 |---------------------------------------------------------------------------------------------------------------------------------------|--------------|--------|-------------|
 | Does your organization lack centralized key and secret management?                                                                    | Yes          | No     | No          |
 | Will you need to limit the creation of keys and secrets to devices to your on-premises hardware, while using these keys in the cloud? | No           | Yes    | No          |
-| Does your organization have rules or policies in place that would prevent keys and secrets from being stored offsite?                | No           | No     | Yes         |
+| Does your organization have rules or policies in place that would prevent keys  from being stored offsite?                | No           | No     | Yes         |
 
 ### Cloud native
 
-With cloud native key management, all keys and secrets are generated, managed, and stored in a cloud-based vault. This approach simplifies many IT tasks related to key management.
+With cloud native key management, all keys and secrets are generated, managed, and stored in a cloud-based vault such as Azure Key Vault. This approach simplifies many IT tasks related to key management, such as key backup, storage, and renewal.
 
-Cloud native key management assumptions: Using a cloud native key management system assumes the following:
+Using a cloud native key management system assumes the following:
 
 - You trust the cloud key management solution with creating, managing, and hosting your organization's secrets and keys.
 - You enable all on-premises applications and services that rely on accessing encryption services or secrets to access the cloud key management system.
 
 ### Hybrid (bring your own key)
 
-With a bring-your-own-key approach, you generate keys on dedicated HSM hardware within your on-premises environment, then transfer the keys to a secure cloud key management system for use with cloud resources.
+With a bring your own key approach, you generate keys on dedicated HSM hardware within your on-premises environment, then transfer the keys to a secure cloud key vault for use with cloud resources.
```

Comment:
I think the hyphens are OK here "bring your own key" modifies "approach".  It's a bit awkward, we could just say "with this approach"

Reply:
Fixed